assignment-webproposal.html

```diff
@@ -38,15 +38,90 @@
+        <li><p>A list of the features of the app, and/or descriptions of all the ways a user can interact with
+          it.  Remember that a web app is not just a static web page with fixed text and pictures; a web
+          app responds to user inputs and actions with sophisticated behaviors.</p></li>
+        <li><p>A list of technical challenges that must be overcome to build the web app.  Keep in mind that
+          no developer can predict technical challenges perfectly in advance, so use your best judgement
+          to guess what will be hard to do.</p></li>
+      </ul>
+      Please include <em>any additional information</em> you feel improves the proposal as well.
+    </p>
     <p>
-      TODO: Fill out
+      Feel free to use any tools you are comfortable with to write your proposal and prepare supporting
+      materials.  The web provides many free tools you can use.  For example,
+      <ul>
+        <li>Google Docs is a great way to prepare a document with text and pictures.</li>
+        <li>Many drawing, sketching, and image editing apps can be helpful in making sketches.</li>
+        <li>Google Image Search can help you find images to incorporate in your designs.</li>
+        <li>There are several free tools available especially for creating mockups of web and mobile apps.</li>
+        <li>Pencil and paper is still a great way to sketch!</li>
+      </ul>
     </p>
-    
+
+    <h2>Presentation</h2>
+    <p>
+      You will present your proposal to a group of your peers during Day 2 of this workshop.  You'll go over
+      your written proposal, and discuss the user story, design, and interactivity of the app.
+      Your peers will ask questions about the proposal, as well as give constructive suggestions for improving
+      the proposal, which you'll discuss as a group.
+    </p>
+    <p>
+      You'll also have the opportunity to listen to your peers' proposals, ask questions about them, and
+      provide feedback with your own ideads of how to improve their proposals.
+    </p>
+
+    <h2>Evaluation</h2>
+    <p>
+      Proposals will be evaluated based on the following.
+      <ul>
+        <li>How thoroughly did you think through your web app's user stories, design, and features?</li>
+        <li>Did you prepare a careful written proposal?</li>
+        <li>How creative is your design?</li>
+        <li>Did you search for, find, and use online resources and tools to improve your proposal?</li>
+      </ul>
+    </p>
+    <p>
+      Presentations will be evaluated based on the following.
+      <ul>
+        <li>Is your presentation well-prepared, clear, and thorough?</li>
+        <li>Do you ask good questions about your peers' proposals?</li>
+        <li>Do listen carefully to questions and do your best to discuss them?</li>
+        <li>Do you provide constructive suggestions about your peers' proposals?</li>
+        <li>Do you accept and adopt your peers' suggestions for your proposal?</li>
```

Comment:
How will we measure this in the 20 minutes? Adopting suggestions seems like a longer term thing.